packages/keychain/src/components/session.tsx

```diff
@@ -75,7 +71,7 @@ export function Session() {
           }
 
           const res = await fetch(url, {
-            body: encodedResponse,
+            body: JSON.stringify(response),
```

Comment:
Will this break slot?

Reply:
i think slot goes through another flow no? the oauth stuff

Reply:
ah yeah what about sozo though? doesn't it use this flow to create a cli controller?

Reply:
ah ok yes that will need to be changed to use the decode the raw json directly. i just wonder if the flow still works because the rev that slot uses for the account_sdk is outdated and when i used it, i had deserialization errors when initializing and using a session account. @glihm do you know if it still works fine?